front/app/containers/Admin/invitations/invite/index.tsx

```diff
@@ -480,333 +396,312 @@ class Invitations extends React.PureComponent<
     return (isValidEmails || isValidInvitationTemplate) && hasValidRights;
   };
 
-  render() {
-    const {
-      projects,
-      locale,
-      tenantLocales,
-      groups,
-      intl: { formatMessage },
-    } = this.props;
-    const {
-      selectedEmails,
-      hasAdminRights,
-      hasModeratorRights,
-      selectedLocale,
-      selectedProjects,
-      selectedGroups,
-      selectedInviteText,
-      invitationOptionsOpened,
-      selectedView,
-      processing,
-      processed,
-      apiErrors,
-      filetypeError,
-      unknownError,
-    } = this.state;
-    const projectOptions = this.getProjectOptions(
-      projects,
-      locale,
-      tenantLocales
-    );
-    const groupOptions = this.getGroupOptions(groups, locale, tenantLocales);
-
-    const invitationTabs = [
-      {
-        name: 'import',
-        label: this.props.intl.formatMessage(messages.importTab),
-      },
-      {
-        name: 'text',
-        label: this.props.intl.formatMessage(messages.textTab),
-      },
-    ];
-
-    const invitationOptions = (
-      <Collapse
-        opened={invitationOptionsOpened}
-        onToggle={this.toggleOptions}
-        label={<FormattedMessage {...messages.invitationOptions} />}
-        labelTooltipText={
-          selectedView === 'import' ? (
-            <FormattedMessage
-              {...messages.importOptionsInfo}
-              values={{
-                supportPageLink: (
-                  <a
-                    href={this.props.intl.formatMessage(
-                      messages.invitesSupportPageURL
-                    )}
-                    target="_blank"
-                    rel="noreferrer"
-                  >
-                    <FormattedMessage {...messages.supportPage} />
-                  </a>
-                ),
-              }}
-            />
-          ) : null
-        }
+  const handleSubmitAction = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (hasAdminRights || hasModeratorRights) {
```

Comment:
Would be great if we can rename these variables. I thought it meant the current user has admin/mod rights.

How about `inviteesWillHaveAdminRights`?